### ISSUE TITLE
Feature suggestions

### ISSUE THREAD

**Issue by YuMao521** [ZalithLauncher/ZalithLauncher#320]
### Feature Description / 功能描述

The launcher has built-in skin change and cape change functions

### This feature request has not been proposed before / 这个功能建议还没有人提出过

- [X] I have checked the Issues and found no similar feature request / 我已经查看过 Issues，并且没有发现与之类似的功能建议被提出

**Comment by Kojoytt** [comment]
Pls add 32 bit update like pojavaluncher pls for 1.20.5 +

**Comment by HarvPlayz** [comment]
Yeah So That Many Players Will Use that Launcher Is Their Main Launcher
